hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/metrics/JRaftMetrics.java

```diff
@@ -65,10 +65,18 @@ public class JRaftMetrics {
     private JRaftMetrics() {
     }
 
-    public synchronized static void init(MeterRegistry meterRegistry) {
+    public synchronized static void init(MeterRegistry meterRegistry, boolean isInitialCall) {
```

Comment:
`isInitialCall` -- Is there an elegant way to avoid this a bit complex branching?